prime-dotnet-webapi/Services/SiteService.cs

```diff
@@ -350,6 +350,17 @@ public async Task<Site> DeclineSite(int siteId)
             return site;
         }
 
+        public async Task<Site> EnableEditingSite(int siteId)
+        {
+            var site = await _context.Sites.SingleOrDefaultAsync(s => s.Id == siteId);
+            site.SubmittedDate = null; ;
+            await _context.SaveChangesAsync();
+
+            await _businessEventService.CreateSiteEventAsync(site.Id, site.Organization.SigningAuthorityId, "Site Enabled Editing");
```

Comment:
I don't feel this is a site creation event, it just seems to be the only business event we have for sites.
I would prefer seeing a new business event for sites; either a generic one or one for status changes. The Approve endpoint should also trigger this new business event.

Reply:
I thought the `CreateSiteEventAsync` does not mean site creation but to 'create' a site event itself. The event description is passed in as parameter "Site Enabled Editing", isn't it?

Reply:
I think you are right, my mistake